integreat_cms/cms/templates/feedback/admin_feedback_list.html

```diff
@@ -54,24 +54,35 @@ <h1 class="heading">{% trans 'Technical Feedback' %}</h1>
                         {% if feedback.object_url %}
                             <a href="{{ feedback.object_url }}" class="hover:underline">{{ feedback.object_name }}</a>
                         {% else %}
-                            {{ feedback.object_name }}
+                        <div class="feedback-entry">
+                            <span class="feedback-entry-content overflow-hidden overflow-ellipsis whitespace-nowrap">
+                                {{ feedback.object_name }}
+                            </span>
+                            <a class="copy-search-btn btn-icon inline" title="{% trans 'Copy to clipboard' %}">
+                                <i icon-name="copy"></i>
+                            </a>
+                            <a class="toggle-feedback-entry">
+                                <i icon-name="chevron-down" class="more transform cursor-pointer hover:scale-125 flex-none"></i>
+                                <i icon-name="chevron-up" class="less transform cursor-pointer hover:scale-125 flex-none"></i>
+                            </a>
+                            </div>
                         {% endif %}
                     </td>
                     <td class="pr-2">{{ feedback.language.translated_name }}</td>
-                    <td class="pr-2 feedback-comment max-w-[75px] xl:max-w-[200px] 2xl:max-w-[300px] 3xl:max-w-[500px] 4xl:max-w-[800px]"
+                    <td class="pr-2 feedback-entry max-w-[75px] xl:max-w-[200px] 2xl:max-w-[300px] 3xl:max-w-[500px] 4xl:max-w-[800px]"
```

Comment:
```suggestion
                    <td class="pr-2 max-w-[75px] xl:max-w-[200px] 2xl:max-w-[300px] 3xl:max-w-[500px] 4xl:max-w-[800px]"
```
I believe the `feedback-entry` is now meant for the child div, not the table cell, right? I think this causes comments in the admin feedback list to overflow when it's too long (in the region feedback list it's working as expected).